v5/api/c/gps.rst

```diff
@@ -0,0 +1,681 @@
+.. highlight:: c
+   :linenothreshold: 5
+
+=========
+GPS C API
+=========
+
+.. note:: For a pros-specific usage guide on the GPS, please check out our article
+          `here <..\tutorials\topical\gps.rst>`_.
+
+.. note::
+   For the C++ API, check out it `here<../cpp/gps.rst>`_.
+
+.. contents:: :local:
+
+Functions
+=========
+
+gps_initialize_full
+-------------------
+
+Set the GPS's offset relative to the center of turning in meters, as well as its initial position.
+
+This function uses the following values of ``errno`` when an error state is reached:
+
+- ``ENXIO`` - The given value is not within the range of V5 ports (1-21).
+- ``ENODEV`` - The port cannot be configured as a GPS.
+- ``EAGAIN``` - The sensor is still calibrating.
+
+.. tabs ::
+   .. tab :: Prototype
+      .. highlight:: c
+      ::
+
+         int32_t gps_initialize_full(uint8_t port, double xInitial, double yInitial, double headingInitial, double xOffset,
+                            double yOffset)
+
+   .. tab :: Example
+      .. highlight:: c
+      ::
+        #define GPS_PORT 1
+        #define X_OFFSET 0
+        #define Y_OFFSET 0
+        #define X_INITIAL 0
+        #define Y_INITIAL 0
+        #define HEADING_INITIAL 0
+
+        void initialize() {
```

Comment:
Example not rendering:
<img width="1140" alt="Screen Shot 2021-08-14 at 2 24 19 PM" src="https://user-images.githubusercontent.com/22580992/129456768-848c2494-1af0-4041-8f8b-48651aff3258.png">

Reply:
Should be fixed

![image](https://user-images.githubusercontent.com/54247087/129667343-86d28cd1-5863-489f-ad01-fdbefd6f809d.png)

Reply:
Though we should change these not to be 0s and 1s...